modules/dataquery/help/dataquery.md

```diff
@@ -1,44 +1,90 @@
-# Data Querying Module
+# Data Query Tool
 
-## Purpose
+The Data Query Tool (DQT) is a simple and easy-to-use interface for querying and exporting data from LORIS.<br>
+<br>
+The DQT module contains the following tabs:<br>
+<br>
+• **Info** : View timestamp of last data update and general information <br>
+• **Define Fields** : Select fields for your query<br>
+• **Define Filters** : Select criteria for your search based on field values<br>
+• **View Data** : See the results of your query<br>
+• **Statistical Analysis** : Visualize data and see basic statistical measures for your query results<br>
+• **Load Saved Query** : Load a previously stored query<br>
 
-The LORIS data querying module ("the DQT") provides a graphical
-frontend for scientists to query and extract arbitrary data from a
-LORIS instances without backend SQL access.
+### Manage Saved Queries
```

Comment:
I missed this one.  it should be formatted like the other bullets above.

Reply:
In your screenshot it looks like `### Define Filters` is not rendering properly - but when I view the file in my browser it looks fine? 

Also, subjectively, I do think the extra line breaks makes it easier to read overall - although I know it's now much longer to scroll through. Hopefully with a new PR I can try to pair down the content so it's not as long.

Reply:
Yup, that's why Help rendering needs to be tested... where it'll be rendered (in the dropdown)
markdown rendering isn't exactly the same everywhere

Reply:
gotcha